src/pipeline.cc

```diff
@@ -1263,6 +1264,7 @@ NAN_METHOD(pipeline) {
   baton->tileSize = AttrTo<uint32_t>(options, "tileSize");
   baton->tileOverlap = AttrTo<uint32_t>(options, "tileOverlap");
   std::string tileContainer = AttrAsStr(options, "tileContainer");
+  baton->tileAngle = AttrTo<uint32_t>(options, "tileAngle");
```

Comment:
The use of `uint32_t` here will discard negative values. `int32_t` should suffice.

Reply:
fixed! Sorry for the headless copy-paste